include/odp/api/spec/queue.h

```diff
@@ -249,7 +251,7 @@ odp_schedule_sync_t odp_queue_sched_type(odp_queue_t queue);
 /**
  * Queue priority
  *
- * @note Passing an invalid queue_handle will result in UNDEFINED behavior
+ * @note The queue must be of type ODP_QUEUE_TYPE_SCHED
```

Comment:
Could use `#ODP_QUEUE_TYPE_SCHED` to enable Doxygen link generation.

Reply:
v9